_includes/pbc/all/install-features/202212.0/marketplace/install-the-merchant-portal-feature.md

```diff
@@ -9,8 +9,8 @@ See [Marketplace Merchant Portal Core feature integration](/docs/pbc/all/merchan
 
 ## Environment requirements
 
-- NodeJs v16+
-- NPM v8+
+- Node.js v16+
+- npm v8+
```

Comment:
"php": ">=8.0" as can be seen [here](https://github.com/spryker-feature/marketplace-merchantportal-core/blob/202212.0/composer.json#LL9C9-L9C24)